tools/UpdateDotnetRuntime.ps1

```diff
@@ -254,6 +254,25 @@ function Update-DevContainer {
     $devContainerDocker | Out-File -FilePath $dockerFilePath -Force
 }
 
+<#
+ .DESCRIPTION Update the DotnetMetadata.json file with the latest version of the SDK
+ #>
+function Update-DotnetRuntimeMetadataChannel {
+    param (
+        [string] $newSdk
+    )
+
+    # -replace uses regex so we are splitting on "."
```

Comment:
This comment doesn't make sense to me.  Why should regex cause us to split on '.'?
```suggestion
    # -replace uses regex so we are splitting on "."
    # Example SDK version: '7.0.100-preview.4' -> '7.0.1xx-preview4'
```